core/modules/node/node.module

```diff
@@ -569,7 +569,7 @@ function node_add_body_field($type, $label = 'Body') {
       'field_name' => 'body',
       'entity_type' => 'node',
       'bundle' => $type->type,
-      'label' => $label,
+      'label' => t($label),
```

Comment:
You cannot translate variables, so you cannot do this. Perhaps try adding the `t()` function in the function declaration? (I have not tested this before though)

Reply:
Search for ` t($` in core yields 81 results.

Reply:
> Search for `t($` in core yields 81 results.

That is sad, if the only thing contained in those `t()` function is a variable 😞 ...translators will literally see "$label", and how are they to translate that?

Reply:
...this is the patch that went in for D9: https://www.drupal.org/files/issues/t-in-fields-is-evil.patch ...I am guessing that there must be a reason why they named that patch that way 😄

Reply:
<img width="468" alt="Screen Shot 2019-09-10 at 4 29 24 am" src="https://user-images.githubusercontent.com/2423362/64556853-9f66c600-d383-11e9-8a27-5d8e1bd760de.png">

...there's more to it: https://api.backdropcms.org/api/backdrop/core%21includes%21bootstrap.inc/function/t/1